src/Datadog.Trace/Tags.cs

```diff
@@ -152,19 +152,7 @@ public static class Tags
         /// <summary>
         /// The sampling priority for the entire trace.
         /// </summary>
-        public const string SamplingPriority = "sampling.priority";
-
-        /// <summary>
-        /// Obsolete. Use <see cref="ManualKeep"/>.
-        /// </summary>
-        [Obsolete("This field will be removed in futures versions of this library. Use ManualKeep instead.")]
-        public const string ForceKeep = "force.keep";
```

Comment:
Can you keep these two constants in the `Datadog.Trace` assembly so there's no breaking change? Or do you also need the tags in the `Datadog.Trace.Core` assembly?

Reply:
There was originally a need to keep them in Core, but with the way I abstracted away the Span storage in the most recent changes, that's not the case anymore.